Readme.md

```diff
@@ -1,4 +1,5 @@
 # E2E tests for console.redhat.com
+[![Playwright Tests](https://github.com/redhat-developer/consoledot-e2e/actions/workflows/playwright.yml/badge.svg?branch=main)](https://github.com/redhat-developer/consoledot-e2e/actions/workflows/playwright.yml)
```

Comment:
NO!

Reply:
would you mind elaborating?

Reply:
@andreaTP all these comments are just for testing of trigger jenins via github PR comment